### PR TITLE
Fix the bug at discord.css font

### PR DESCRIPTION
**Pull request by no1crate**

### CODE REVIEW

File: themes/dark/discord.css [hunk 1]
```diff
@@ -18,7 +18,7 @@ body {
 	--roundness: 1; /* How round is this theme? Applied automatically to borders. Unitless */
 
 	/* Fonts! */
-	font-family: PTSans, sans-serif;
+	font-family: '', sans-serif;
```
Comment (kurmachu) [inline themes/dark/discord.css:21]: You cannot simply omit a font...?

**Review by kurmachu — CHANGES_REQUESTED** [review]

File: themes/dark/discord.css (hunk 1 above)
Comment (no1crate) [reply themes/dark/discord.css:21]: It's works for me, and shows the font that I want.

Comment (no1crate) [reply themes/dark/discord.css:21]: Or i'm going to do something for that

**Comment by no1crate** [comment]
Fixed

**Review by kurmachu — APPROVED** [review]
Times New Roman? Urgh.

**Comment by no1crate** [comment]
> Times New Roman? Urgh.

I dont know, i searched what font opera uses and put that there

**Pull request merged**